src/components/App.tsx

```diff
@@ -133,6 +134,38 @@ export default function App(): ReactElement {
     return JSON5.parse<Config>(state.configInput);
   };
 
+  const onImportButton = () => {
+    document.getElementById("import-input")?.click();
+  };
```

Comment:
This is my alternative to the previous `<label htmlFor="import-input">` way. I'm not sure it's any less hacky, but it at least doesn't need to re-implement pieces of `ArrowButton` as a non-button anymore.